src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/cronjob/SykmeldingNyCronjob.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.syfo.oppfolgingstilfelle.bit.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.cronjob.Cronjob
+import no.nav.syfo.application.cronjob.CronjobResult
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.client.arbeidsforhold.ArbeidsforholdClient
+import no.nav.syfo.oppfolgingstilfelle.bit.database.*
+import no.nav.syfo.oppfolgingstilfelle.bit.database.domain.toOppfolgingstilfelleBitList
+import org.slf4j.LoggerFactory
+
+class SykmeldingNyCronjob(
+    private val database: DatabaseInterface,
+    private val arbeidsforholdClient: ArbeidsforholdClient,
+) : Cronjob {
+    override val initialDelayMinutes: Long = 7
```

Comment:
Den andre cronjob'en her initialDelay=2, så tanken er at de to crojob'ene ikke skal kjøre samtidig (selv om det også antakelig vil gå bra).